scheduler/src/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java

```diff
@@ -12,7 +12,7 @@
  * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  * See the GNU Lesser General Public License for more details.
  *
- * Copyright (c) 2002-2013 Pentaho Corporation..  All rights reserved.
+ * Copyright (c) 2002-2017 Pentaho Corporation..  All rights reserved.
```

Comment:
#3398 and #3397 didn't update QuartzScheduler but this one does. Seems like a regression waiting to happen.